### DISCUSSION TITLE
How do I add cdn imports for my component?

### DISCUSSION THREAD

**Discussion by simkimsia** [django-components/django-components#158] [Q&A] [answered]
I finished the readme and did the calendar example successfully. Thank you for writing that example up 🙏 

I found a couple of typo and submitted the correction as [PR](https://github.com/EmilStenstrom/django-components/pull/157).

I am now dipping my toes into building my own component from an opensource javascript library, tagify

eg. in https://github.com/yairEO/tagify#option-1---import-from-cdn 

i need to import 

```
<script src="https://unpkg.com/@yaireo/tagify"></script>
<script src="https://unpkg.com/@yaireo/tagify/dist/tagify.polyfills.min.js"></script>
<link href="https://unpkg.com/@yaireo/tagify/dist/tagify.css" rel="stylesheet" type="text/css" />
```

How do I import them? I cannot use the `class Media` right?

I tried understanding `MediaDefiningClass` in https://github.com/django/django/blob/d0863429a39dbc99a52a6d781006e7075c743d33/django/forms/widgets.py but I don't quite grok it.

And also https://github.com/EmilStenstrom/django-components/blob/87e2889cdf818791ebaf572e9b578732edb74f1c/django_components/component.py#L24

**Comment by EmilStenstrom** [comment, marked as answer]
Hi! Reading the [Form Asset docs from django](https://docs.djangoproject.com/en/4.0/topics/forms/media/#paths-in-asset-definitions), it looks like your should just be able to add paths that start with http/https there? What happens if you try?

**Reply by simkimsia** [reply]
They work thanks! I now have a separate issue. I will write a separate discussion
